main.go

```diff
@@ -138,6 +138,12 @@ func main() {
 		updateCommand,
 	}
 	app.Before = func(context *cli.Context) error {
+
+		// do nothing if logrus was already initialized in init.go
+		if logrus.StandardLogger().Out != logrus.New().Out {
```

Comment:
I'm not really a fan of this -- why not set up logging for `init` here (or setting a global flag) rather than doing it this way?